### ISSUE TITLE
Consistent Y axis scales on nested charts.

### ISSUE THREAD

**Issue by ruthpozuelo** [microsoft/charticulator#940]
**Is your feature request related to a problem? Please describe.**
When I create a nested chart based on area charts (glyph as a rectangle), as there is no Y-axis, I get different scales on the nested charts. Is there a way to force charticulator to use the same Y-axis across nested charts?
![image](https://user-images.githubusercontent.com/19476550/155972537-4a278399-b894-43b9-aa3e-a34a071c208e.png)

I have tried to do it with a line chart too where I can specify a Y axis, but I am having problems with the auto-update feature. I will post that issue on an issue request.

**Comment by bongshin** [comment]
This is a duplicate of https://github.com/microsoft/charticulator/issues/800.

**Comment by jordanFromNYC** [comment]
Same applies to Multi-glyph charts (not nested). The Y axis are not aligning

**Comment by ruthpozuelo** [comment]
Have you tried to set the scale manually on both glyphs? By default they will adjust to the max and min of the values, but you can hardcode it to anything you want.

**Comment by jordanFromNYC** [comment]
Thing is my data is quite dynamic and can vary a lot depending on the filters, i need the axis to adjust to the right scale

**Comment by ruthpozuelo** [comment]
Yes, in that case, I am with you. It would be lovely if we could have a setting to sync the axis between glyps. Maybe you should open a new feature request to make sure it doesnt get lost in the comments section?

**Comment by jordanFromNYC** [comment]
That said, i noticed that multi glyph bar charts have synced Y axis, while symbols don't; seems to be due tot he fact that for bars the axis is added via 'add a legend' and seems this legend is shared by all the glyphs.
Opening a new request

**Comment by milkahut** [comment]
Is there any work around to sync the y axis for symbols and charts in multi glyphs?
